azerothcore/azerothcore-wotlk
# fix(DB/npc_vendor): Empty vendors honor/arena S6 horde side

##### CHANGES PROPOSED:
This is basically the PR from @solidmaxtor, which sadly was opened multiple times incorrectly:
#1946 
#1949 
In order to not loose those changes I condensed the SQL script and provide this PR in the correct format. I hope this is ok for you @solidmaxtor. If not I can close this PR.

###### ISSUES ADDRESSED:
Closes #1944 

##### TESTS PERFORMED:
just tested if the SQL script can be applied without error, I did _not_ test this in-game

##### HOW TO TEST THE CHANGES:
Copied from #1949:

change: Arena.ArenaSeason.ID = 6
restar server and check npc horde in halls of leyends - Orgrimar

OR check npc_vendor in world db table
- Sergeant Thunderhorn id 34036 vendor items list in season 6
- Blood Guard Zar'shi id 34061 vendor items list in season 6
- Doris Volanthius id 34058 vendor items list in season 6

##### KNOWN ISSUES AND TODO LIST:
none

##### Target branch(es):
Master
